src/utils/institutions-helpers.ts

```diff
@@ -20,3 +21,24 @@ export const getMostSpecificUnit = (values: FormikInstitutionUnit): InstitutionU
   }
   return values as InstitutionUnitBase;
 };
+
+// Find distinct unit URIs for a set of contributors' affiliations
+export const getDistinctContributorUnits = (contributors: Contributor[]) => [
+  ...new Set(
+    contributors
+      .map((contributor) => contributor.affiliations)
+      .flat()
+      .filter((unit) => unit)
+      .map((unit) => unit.id)
+  ),
+];
+
+// Returns top-down comma separated unit names
+export const getCommaSeparatedUnitString = (unit: RecursiveInstitutionUnit, unitNames: string[] = []): string => {
```

Comment:
er det ikke kun parent unit som skal vises? eller skal alle forskjellige institutt og avdelinger også vises?

Reply:
hele regla, ja

Reply:
type "NTNU, Faktultet for Ost, Osteinstituttet"